src/atomist.config.ts

```diff
@@ -29,6 +29,8 @@ const machineOptions: ConfigureOptions = {
         "sdm.docker.hub.registry",
         "sdm.docker.hub.user",
         "sdm.docker.hub.password",
+        "sdm.generator.spring.project.owner",
+        "sdm.generator.spring.project.repo",
```

Comment:
Rename to `sdm.seed.spring.owner` and `sdm.seed.spring.repo`. Also please make them optional and default to a _good_ seed repo.

Reply:
Currently using spring-petclinic, but there's a bug that's breaking the package renaming (usage of the word `package` in a top comment)